### PR TITLE
change throw to console.warn

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: packages/plugin/iframe/src/lib/connector.ts [hunk 1]
```diff
@@ -13,6 +13,9 @@ import {
 
 import { listenOnThemeChanged } from './theme'
 
+function warn() {
```
Comment (yann300) [inline packages/plugin/iframe/src/lib/connector.ts:16]: is that needed?

Comment (GrandSchtroumpf) [reply packages/plugin/iframe/src/lib/connector.ts:16]: ah sorry, I forgot to remove it

File: packages/plugin/iframe/src/lib/connector.ts [hunk 2]
```diff
@@ -32,11 +35,11 @@ export class IframeConnector implements ClientConnector {
   /** Get messae from the engine */
   on(cb: (message: Partial<Message>) => void) {
     window.addEventListener('message', async (event: MessageEvent) => {
-      if (!event.source) throw new Error('No source')
+      if (!event.source) return console.warn('No source in message', event)
+      if (!event.data) return console.warn('No data provided in message', event)
```
Comment (yann300) [inline packages/plugin/iframe/src/lib/connector.ts:39]: At this point we should probably also filter messages that are supposedly not for us.
e.g which doesn't have an `action` property (and also other required properties)

Comment (GrandSchtroumpf) [reply packages/plugin/iframe/src/lib/connector.ts:36]: Alright

**Review by yann300 — APPROVED** [review]

**Pull request merged**